src/modules/awakenMember.ts

```diff
@@ -16,7 +16,9 @@ export const awakenMember = async (bot: ExtendedClient) => {
     if (!guild || !awakenedRole) {
       return;
     }
-    const members = await guild.members.fetch();
+    const members = (await guild.members.fetch()).filter(
+      (guildMember) => guildMember.user.bot === false
```

Comment:
```suggestion
      (guildMember) => !guildMember.user.bot
```

May need to be reformatted